db/migrate/20210910085912_migrate_pin_payment_to_cash.rb

```diff
@@ -0,0 +1,5 @@
+class MigratePinPaymentToCash < ActiveRecord::Migration[6.1]
+  def change
+    execute "UPDATE spree_payment_methods SET type = 'Spree::PaymentMethod::Check' WHERE type = 'Spree::Gatewat::Pin'"
```

Comment:
`Spree::Gatewat::Pin`

Looks like a typo :)

Reply:
As part of the test we should probably verify on a staging server that there is at least one of payment method where type = `'Spree::Gateway::Pin'` before running this migration, and that there are none after running it.

Reply:
> As part of the test we should probably verify on a staging server that there is at least one of payment method where type = `'Spree::Gateway::Pin'` before running this migration, and that there are none after running it.

I've had a chat with @filipefurtad0 and it seems only staging FR has pin payments, and it's not operational at the moment. So we'll have to wait a but until it can be tested.